### ISSUE TITLE
Plugin removed from RuneLite and plugin hub

### ISSUE THREAD

**Issue by Hakej** [RKGman/fire-beats#6]
So I installed the plugin yesterday and it was really buggy like it was not working at all. Volume change seemed to not matter at all, base music when there was no remix never played and the only actual remix that played was the remix of the location where I was when I turned the plugin on. (the music never changed)

Now I can't find it anywhere, neither on plugin hub and on my RuneLite, so here's my question - was this plugin just removed, or maybe some bigger bug has occurred with it on my side?

**Comment by RKGman** [comment]
Looking into this now.  Not sure how quick of a fix it will be, as I'll be patching some other things in the process.  Thanks for the info!

Also, to answer your question, the plugin was removed because of some dependency issues with RuneLite update, which would explain the plugin not doing anything.

However here are a few notes:

- Volume control only applies from the control panel (when you click on the fire beats icon and it brings up the menu within RuneLite, not the OSRS client).  

- The music not playing usually happens when anonfile servers are being slow to deliver files, which we have no control over.  

Maybe one day I will be able to leverage a more reliable streaming service like SoundCloud, but this is the best we've got for now.
